### PR TITLE
adding queue logger for multiprocess logging

### PR DESCRIPTION
**Pull request by goodenou**
trying a new pull request. not sure why tests are failing.

### CODE REVIEW

**Comment by jcpunk** [comment]
There seems to be something weird with the stop methods causing the unit tests to fail.... I've not figured out what is wrong, but it seems localized to there.

File: src/decisionengine/framework/modules/de_logger.py [hunk 1]
```diff
@@ -55,79 +51,70 @@ def set_logging(
...
+    delogger.debug("de logging setup complete")
 
 
 def get_logger():
     """
     get default logger - "decisionengine"
     :rtype: :class:`logging.Logger` - rotating file logger
     """
-    return logger
+    return delogger
```
Comment (jcpunk) [inline src/decisionengine/framework/modules/de_logger.py:105]: I believe this should be :

```suggestion
    nonlocal delogger
    return delogger
```

File: src/decisionengine/framework/modules/de_logger.py [hunk 2]
```diff
@@ -55,79 +51,70 @@ def set_logging(
...
+
+
+def get_queue_logger():
+    """
+    get QueueLogger which owns the logging queues and listeners
+    :rtype: :class:`decisionengine.framework.modules.QueueLogger``
+    """
+    return queue_logger
```
Comment (jcpunk) [inline src/decisionengine/framework/modules/de_logger.py:113]: Shouldn't this be:

```suggestion
    nonlocal queue_logger
    return queue_logger
```

File: src/decisionengine/framework/modules/de_logger.py [hunk 3]
```diff
@@ -55,79 +51,70 @@ def set_logging(
...
+
+    # setup the queue logger
+    if unit_testing is False:
+        queue_logger.setup_queue_logging(delogger, structlog_handlers_list)
+        queue_logger.start()
+
+    delogger.debug("de logging setup complete")
 
 
 def get_logger():
```
Comment (jcpunk) [inline src/decisionengine/framework/modules/de_logger.py:100]: Do we have unit tests making sure this is returning the right value?

File: src/decisionengine/framework/modules/de_logger.py [hunk 4]
```diff
@@ -55,79 +51,70 @@ def set_logging(
...
     """
     get default logger - "decisionengine"
     :rtype: :class:`logging.Logger` - rotating file logger
     """
-    return logger
+    return delogger
+
+
+def get_queue_logger():
```
Comment (jcpunk) [inline src/decisionengine/framework/modules/de_logger.py:108]: Do we have unit tests making sure this is returning the right value?

**Comment by goodenou** [comment]
Issues with unit tests, were, I believe, related to the creation of a queue logger which was not used and shut down properly in the tests. I have added a flag for not creating the queue logger in the unit tests, and have put that code in a new PR.

**Pull request closed without merging**